Gemfile

```diff
@@ -30,4 +33,12 @@ group :development do
   gem 'web-console', '>= 3.3.0'
 end
 
+group :test do
```

Comment:
I see you add them here as well. Why?????

Reply:
@vlodkopits

Reply:
No, i don't. This gem is present in develop branch, however my branch wasn't merged with develop. Only >>>>'gem 'webpacker', git: 'https://github.com/rails/webpacker.git'<<< this line of code was added by me.

Reply:
Do you what is rebase?

Reply:
Yes, u are right. Changes from develop branch was merged* to this one without, and not rebased. That is the cause and the only reason. Further, i will only use git rebase. Thank you)